modelplugin/aether-1.0.0/yang/aether-profile@2020-07-15.yang

```diff
@@ -0,0 +1,49 @@
+module aether-profile {
```

Comment:
I wonder if the name of the YANG file should be more specific e.g. `subscriber profile` or something. We might need to have other types of profiles on other things